src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+            'CSV Field Mapping'                   : 'Attribuer des champs CSV',
+            'Preview Line'                        : 'Aperçus de la première ligne',
+            'Line {line}'                         : 'Ligne {line}',
+            'Ignore'                              : 'Ignorer',
+            'Changed'                             : 'Changé',
+            'Modified'                            : 'Modifié',
+            'Edited'                              : 'Édité',
+            'Created'                             : 'Crée',
+            'Folder'                              : 'Dossier',
+            'Revision'                            : 'Révision',
+            'Url'                                 : 'Lien',
+            'Parent'                              : 'Parent',
+            'Run Import'                          : 'Lancer l’import',
+            'Import'                              : 'Importer',
+            'Parsing input file'                  : 'Analyse du fichier téléversé',
+            'Analyzing tags'                      : 'Analyse des étiquettes',
+            'Analyzing folders'                   : 'Analyse des dossiers',
+            'Reading tags'                        : 'Lecture des étiquettes',
+            'Importing tags'                      : 'Import des étiquettes',
+            'Reading folders'                     : 'Lecture des dossiers',
+            'Importing folders'                   : 'Import des dossiers',
+            'Reading passwords'                   : 'Lecture des mots de passe',
+            'Importing passwords'                 : 'Import des mots de passe',
+            'Import failed'                       : 'Échec de l’import',
+            'Import partially failed'             : 'Import partiellement réussi',
+            'Import successful'                   : 'Import réussi',
+            'Import error'                        : 'Erreur lors de l’import',
+            'Invalid file type "{type}"'          : 'Format de ficher invalide "{type}"',
+            'Password required'                   : 'Un mot de passe requis',
+            'Password invalid'                    : 'Mot de passe erroné',
+            'Failed to decrypt passwords'         : 'Déchiffrement des mots de passé échoué',
+            'Failed to decrypt folders'           : 'Déchiffrement des dossiers échoué',
+            'Failed to decrypt tags'              : 'Déchiffrement des étiquettes échoué',
+            'CSV file can not be mapped'          : 'Le fichier CSV ne peut pas être cartographié',
+            'Delete All Items'                    : 'Effacer tous les éléments',
+            'Restore All Items'                   : 'Restaurer tous les éléments',
+            'Restore Items'                       : 'Restaure les éléments',
+            'Restore all items in trash?'         : 'Restaurer les éléments à leurs emplacement initiaux',
```

Comment:
I think there is a mistake here. Transalation basic says "_Restore elements to initial location_" which is not what seems to be expected.
Should be `Restaurer les éléments dans la corbeille`